api/swagger.yml

```diff
@@ -274,6 +274,19 @@ components:
           items:
             $ref: "#/components/schemas/ObjectStats"
 
+    ObjectCopyCreation:
+      type: object
+      required:
+        - source
+        - destination
+      properties:
+        source:
+          type: string
+          description: path of the copied object relative to the branch
+        destination:
+          type: string
+          description: the new copy path relative to the branch
```

Comment:
Why can't I use this API to copy to another branch?

Reply:
No where did we discuss about enabling copy between branches.
Do we really want to support this?

Reply:
How can we not? The StageObject API does not contain this limitation right? Shouldn't we support the same use cases?
We can "really" copy if not the same branch, and "fancy-link" if it is. Also - the source may be any kind of ref, like tag or commit.
I understand this is just an initial PR focused on unblocking, so copying everything for now is fine.

Reply:
Decided to support only in-branch copy, and take cross-branch copy to discussion